src/Arcus.Testing.Logging.MSTest/Extensions/ILoggerBuilderExtensions.cs

```diff
@@ -27,14 +27,9 @@ public static ILoggingBuilder AddMSTestLogging(this ILoggingBuilder builder, Tes
         }
 
         [ProviderAlias("MSTest")]
-        private sealed class MSTestLoggerProvider : ILoggerProvider
+        private sealed class MSTestLoggerProvider(TestContext context) : ILoggerProvider
```

Comment:
Never thought you're going to apply those 'primary constructors'.
Please be aware that the scope of the `context` parameter is now the entire class definition.

That's one of the reasons why I'm personally not a fan of them.  But it might be me.

Reply:
For these small private classes that mostly act as adapter classes, they might be of use to limit code. They now look a bit like how classes are defined in F#.

Reply:
It's OK for me, I was just a bit surprised.  But I've should've known. :)
Anyway, personally I don't like them (because of the 'unclear' scope and the possibility that you unintendly use / capture that constructor parameter.  
But I see why you'd use them, and I see the benefit of it in simple classes.